client/views/chan.tpl

```diff
@@ -1,7 +1,7 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
-	<button class="close" aria-label="Close"></button>
+	{{#equal type "lobby"}}{{else}}<button class="close" aria-label="Close"></button>{{/equal}}
```

Comment:
Euch, can we drop in another handlebars helper that is the opposite of equal so we don't have to have this weird thing?

Reply:
@YaManicKill, agreed, but do you mind if we merge as-is instead of asking @Cldfire to add a Handlebars helper for that?
If you're okay with that, then after we merge I'll add such helper with tests for it, also add tests for `equal`. I can do that today, right after merging.

I thought `diff` was meant for that but it's actually for... something else...
At some point, we might want to use something like https://github.com/helpers/handlebars-helpers but right now we don't need it so we can look into that when we start adding more generic helpers. Also I couldn't make it work with Webpack lol.

Reply:
@YaManicKill, done at #1874. Whichever PR goes first, I'll adapt the other accordingly. Feel free to review/approve both and I'll remember to do the conversion before merging.

Reply:
Alrighty